exec/java-exec/src/main/java/org/apache/drill/exec/vector/BitVector.java

```diff
@@ -211,8 +220,11 @@ public TransferPair makeTransferPair(ValueVector to) {
 
   public void transferTo(BitVector target) {
     target.clear();
+    if (target.data != null) {
+      target.data.release();
```

Comment:
release(1) ?  

Also, do you intend to change from retain() to retain(1) across the source codes? Seems some places are still using retain(). Do a quick search will find couple of places, including BitVector.load(), VariableLengthVectors.java:load() etc.